### DISCUSSION TITLE
ownership/intro

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#199] [Book Comments]
# ownership/intro

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/ownership/intro.html

**Comment by vickyone0** [comment]
lets get start

**Reply by CamaroW** [reply]
Let's get started

**Comment by rahul-real** [comment]
Nice Concept

**Comment by nhalubai** [comment]
Nice one

**Comment by vickyone0** [comment]
done
